task-11/task-11/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Task_1
+{
+    /// <summary>
+    /// The Class EntryPoint provides main
+    /// </summary>
+    /// <param name="args">arguments from command line</param>
+    class EntryPoint
+    {
+        static void Main(string[] args)
```

Comment:
please add summary for the method

Reply:
i have some problems with repository. So IgorKheidorov said to creat new one and add files there. it is about  Commits on Mar 19, 2019
Deadline=03/10/2019 18-00
?